views/apartments/index.erb

```diff
@@ -3,3 +3,7 @@
   <li><a href='/apartments/2'>Apartment 2</a></li>
   <li><a href='/apartments/3'>Apartment 3</a></li>
 </ul>
+
+<li>
+   <% @apartments.each do |apartment| %>
+</li>
```

Comment:
I would look into the [ruby enumerable docs](http://ruby-doc.org/core-2.2.2/Enumerable.html) to get a closer look at ruby enumerables